com.unity.render-pipelines.universal/Shaders/Utils/Sampling.shader

```diff
@@ -1,53 +1,5 @@
-Shader "Hidden/Universal Render Pipeline/Sampling"
```

Comment:
This might cause breaking changes, if user is doing Shader.Find(), I'm not sure.
If we change the name of the shader here we should also change the name of the file.